Samples/PhotoEditor/cpp-winui-packaged/PhotoEditor/Package.appxmanifest

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="96eb8b1a-8885-4cd2-ab4d-8f31ffc46bc7"
+    Publisher="CN=stwhi"
```

Comment:
Guidelines are to set Publisher and PublisherDisplayName to "Microsoft Corporation" - see the [template Package.appxmanifest](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/Templates/VSIX/Project%20Templates/cpp-winui-template/Package.appxmanifest) as an example

Reply:
Fixed